.github/workflows/ci.yml

```diff
@@ -0,0 +1,122 @@
+name: Build and push
+
+on:
+  push:
+  pull_request:
```

Comment:
we could let the workflow run every week automatically to produce updated images (e.g. if Alpine updates).

Reply:
Cool, that sounds like a good idea.  I'd been wondering how other projects handle that. :smile:

Reply:
addressed with 012887ff.

Reply:
> Cool, that sounds like a good idea. I'd been wondering how other projects handle that. 😄

with daily builds. to keep the base system up to date you have to build the container every day/week with updates in the dockerfile

Reply:
Cool.  Sounds like we just need to set up some scheduler somewhere and then mostly forget about it. :smile: